mlir/tools/mlir-miopen-lib/CMakeLists.txt

```diff
@@ -75,9 +73,14 @@ if( NOT BUILD_SHARED_LIBS )
     add_custom_command(TARGET ${output_archive}
                        POST_BUILD
                        COMMAND ${CMAKE_AR} -M < ${mri_file}
-                       COMMAND ${CMAKE_COMMAND} -E copy ${full_output_path} ${install_path}
-                       COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_SOURCE_DIR}/Miir.h ${header_path}
                        DEPENDS ${output_archive_dummy_file})
+
+    install(FILES ${full_output_path}
+            TYPE LIB
+            COMPONENT libMLIRMIOpen EXCLUDE_FROM_ALL)
+    install(FILES Miir.h
+            TYPE INCLUDE
+            COMPONENT libMLIRMIOpen EXCLUDE_FROM_ALL)
```

Comment:
[Question] Just want to confirm that there's no change of behavior here. Building the target still install the library?

Reply:
There is a change of behavior. Building the target now does *not* install the library, hence the calls in the Jenkins file to `cmake --install`

Reply:
We synced offline, the new build install sequence is:

```bash
cmake -G "Unix Makefiles" .. -DCMAKE_BUILD_TYPE=Release -DBUILD_FAT_LIBMLIRMIOPEN=1
make -j libMLIRMIOpen
cmake --install . --component libMLIRMIOpen
```

Reply:
You don't necessarily need to call for Makefiles - you could also, say, call for Ninja, or just leave `-G` off and let it default to Makefiles

Reply:
> or just leave -G off and let it default to Makefiles

Thanks for the suggestion